openedx/features/colaraz_features/helpers.py

```diff
@@ -427,3 +431,16 @@ def get_query_dict(_dict):
         else:
             q[key] = value
     return q
+
+
+def get_role_based_urls(response):
+    email = response.get('email')
+    access_token = response.get('access_token')
+    if email and access_token and hasattr(settings, 'COLARAZ_APP_LINKS_API'):
+        api_url = '{}?{}'.format(getattr(settings, 'COLARAZ_APP_LINKS_API'), urlencode({'email': email}))
+        resp = requests.get(api_url, headers={'Authorization': 'Bearer {}'.format(access_token)})
+        if resp.status_code == 200:
+            return json.loads(resp.content)
+
+    LOGGER.error('Colaraz application urls api is not working appropriately')
```

Comment:
What if we log different errors for cases when `COLARAZ_APP_LINKS_API` is not present in the configs and when the API is actually not working properly,

Reply:
updated